sycl/test/graph/graph-explicit-dotp.cpp

```diff
@@ -83,16 +83,15 @@ int main() {
   // Using shortcut for executing a graph of commands
   q.ext_oneapi_graph(executable_graph).wait();
 
-  if (*dotp != host_gold_result()) {
-    std::cout << "Error unexpected result!\n";
-  }
+  if (*dotp == host_gold_result())
+    std::cout << "Dot product explicit graph test passed." << std::endl;
```

Comment:
We might want to stick to generic messages, instead of test specific. Both for test and fail. Which simplifies automated testing.